karma.conf.js

```diff
@@ -19,7 +19,9 @@ module.exports = function(config) {
       'https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.3.1/angular-ui-router.js',
       'https://www.gstatic.com/firebasejs/3.0.4/firebase.js',
       'https://cdnjs.cloudflare.com/ajax/libs/angularFire/2.0.1/angularfire.js',
-      'app/**/*.spec.js'
+      'node_modules/angular-mocks/angular-mocks.js',
+      'app/**/*.js',
+      'test/**/*.spec.js'
```

Comment:
@sbaughman, I'm a little stuck here but I should be able to figure it out soon. Basically what happens is:

1. Run `karma start`
2. Karma loads up all our project files in Chrome
3. Karma runs all the tests in Chrome
4. The terminal outputs the result

_**Issue:** I think the issue I'm having is that the project files aren't loading in any specific order so it throws a totally nondescript error. The quick work around is to just do it all manually. I will probably try that later tonight._